helm-charts/0.23.1/values.yaml

```diff
@@ -13,7 +13,7 @@ global:
   # possible options for baseImage (distroless, distroless-fips, photon, photon-fips)
   baseImage: distroless
   registry: vsecm
-  logLevel: "3"
+  logLevel: "7"
```

Comment:
log level changed; but that’s fine.

I’m planning to change all to 7. 

That’s what SPIFFE/SPIRE does anyway.